tutorials/introduction-gee-python-api-guiattard/tutorial.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<ul><li>Section title and throughout: geotiff &gt; GeoTIFF</li><li>Lowercase throughout: Image &gt; image</li><li>Sp. personnal &gt; personal</li><li>Sp. georeferecend &gt; georeferenced</li><li>Remove "the": in <strong>the</strong> Google Drive &gt; in Google Drive</li><li>Link to the Developer Guide section on exporting: "There are multiple ways to do that (see the [Exporting](https://developers.google.com/earth-engine/guides/exporting#exporting-images) section of the Developer Guide)."</li><li>Would be helpful to let people know that the file is exported to their Google Drive directory head and named according to the <em>fileNamePrefix</em> parameter</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/254/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>Agree. The text becomes:</p><p> </p><blockquote>Export a GeoTIFF file</blockquote><blockquote>After manipulating Earth Engine datasets, you may need to export a resulting Image to a GeoTIFF. For example, to use it as an input of a numerical model, or to overlap it with personal georeferencend files in your favorite GIS. There are multiple ways to do that (see the&nbsp;<a href="https://developers.google.com/earth-engine/guides/exporting#exporting-images" target="_blank">Exporting</a>&nbsp;section of the Developer Guide). Here we explore two:</blockquote><blockquote>In the first option, we save the Image you want in Google Drive,</blockquote><blockquote>In the second option, we directly download the image.</blockquote><blockquote>Save a GeoTIFF file in your Google Drive</blockquote><blockquote>To export the image to our Google Drive, we have to define a task and start it. We have to specify the size of pixels (here 30 m), the projection (here EPSG:4326), the file format (here GeoTIFF), the region of interest (here the area of lyon defined before) and the file will be exported to our Google Drive directory head and named according to the&nbsp;<em>fileNamePrefix</em>&nbsp;we choose.</blockquote>